src/plugin/systems/mod.rs

```diff
@@ -342,24 +331,4 @@ pub mod tests {
             approx::assert_relative_eq!(body_transform.scale, child_transform.scale,);
         }
     }
-
-    // Allows run tests for systems containing rendering related things without GPU
-    pub struct HeadlessRenderPlugin;
-
-    impl Plugin for HeadlessRenderPlugin {
-        fn build(&self, app: &mut App) {
-            app.add_plugins((
-                AssetPlugin::default(),
-                ScenePlugin,
-                RenderPlugin {
-                    render_creation: RenderCreation::Automatic(WgpuSettings {
-                        backends: None,
-                        ..Default::default()
-                    }),
-                    ..Default::default()
-                },
```

Comment:
- `RenderPlugin` seems indeed unnecessarry, but the feature `async-collider` still needs `AssetPlugin` + `MeshPlugin` + `ScenePlugin` ; see https://github.com/dimforge/bevy_rapier/pull/625

Reply:
I removed that commit impacting headless plugin, the issue linked above should be merged first.